data/datapackage.yaml

```diff
@@ -0,0 +1,338 @@
+# --------
+# metadata: annotations.csv references.csv library.csv insertions.csv
+# --------
+
+profile: data-package
+resources:
+  - encoding: utf-8-sig
+    format: csv
+    hashing: md5
+    name: annotations
+    path: annotations.csv
+    profile: tabular-data-resource
+    description: A collection of all written annotations in Jacques Derrida's library
+    schema:
+      primaryKey: id
+      foreignKeys:
+        - fields: book_id
+          reference: 
+            resource: library
+            fields: id
+      fields:
```

Comment:
Did you generate these descriptions pulled from the model code, or just make them up yourself?

(Now wondering how we could automate that part of generating the datapackage, but trying to tell my brain to stop that! 🧠 🛑 )

Reply:
These were just me guessing at what the fields meant. It seemed faster to have you correct me? 🤷‍♀️ I realize that wasn't the safest way to do this though

Reply:
not sure what is the most efficient way to edit/revise them — pull request doesn't seem like it. google doc? (but then you'll have to put the changes back in ☹️ )